spyder/config/main.py

```diff
@@ -54,6 +54,12 @@
 MAC = sys.platform == 'darwin'
 CTRL = "Meta" if MAC else "Ctrl"
 
+# Run cell shortcuts
+if sys.platform == 'darwin':
+    RUN_CELL_SHORTCUT = 'Meta+Enter'
+else:
+    RUN_CELL_SHORTCUT = 'Ctrl+Enter'
+RUN_CELL_AND_ADVANCE_SHORTCUT = 'Shift+Enter'

```

Comment:
In some platforms they aren't equivalent, some keyboards layouts have a different key for enter and return https://en.wikipedia.org/wiki/Enter_key#Differences_between_Enter_and_Return

I'll change 'Enter' to 'Return'